source/concepts/networking/network-firewall.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+owner_slack: "#modernisation-platform"
+title: AWS Network Firewall
+last_reviewed_on: 2022-02-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+We currently have setup in our core-network account AWS Network Firewall to monitor and control network traffic on a layer 3 level for transit gateway connection and cross account vpcs to protecte reources
+such as EC2 instances, along with a general aws resources
+
+## What is an AWS network firewall
+
+AWS Network Firewall is a managed service that makes it easy to deploy essential network protections for all of your Amazon Virtual Private Clouds (VPCs). The service can be setup with just a few clicks and scales automatically with your network traffic, so you don't have to worry about deploying and managing any infrastructure. AWS Network Firewall’s flexible rules engine lets you define firewall rules that give you fine-grained control over network traffic, such as blocking outbound Server Message Block (SMB) requests to prevent the spread of malicious activity. You can also import rules you’ve already written in common open source rule formats as well as enable integrations with managed intelligence feeds sourced by AWS partners. AWS Network Firewall works together with AWS Firewall Manager so you can build policies based on AWS Network Firewall rules and then centrally apply those policies across your VPCs and accounts.
+
+## Amending Rules
+
+Rules can be amended by creating a pull request against the modernisation-platform repo and amending one of the following four file names
+development_rules.json
+test_rules.json
+preproduction_rules.json
+production_rules.json
+
+The rules files are broken down to environment and not account to make thing easier keep track of, to this end i have included a link to the MP cid
+
+[CIDR Ranges](https://github.com/ministryofjustice/modernisation-platform/blob/main/cidr-allocation.md)
+
+## Example
+
+Below is an example of PR of how add rules. In this example there is 1 additional rule to be added for the network firewall.
```

Comment:
`how to add rules` instead of `how add rules`. Also there's no example in the document.

Reply:
example pr added as a hyper link to the bottom of the document